sakip_sabanci_config.rb

```diff
@@ -27,10 +27,10 @@
 to_field 'cho_description', extract_oai('dc:description'), strip
 to_field 'cho_dc_rights', extract_oai('dc:rights'), strip
 to_field 'cho_edm_type', extract_oai('dc:type'),
-         strip, translation_map('not_found', 'types')
+         transform(strip: true, downcase: true), translation_map('not_found', 'types')
 to_field 'cho_format', extract_oai('dc:format'), strip
 to_field 'cho_language', extract_oai('dc:language'), split(';'),
-         strip, translation_map('not_found', 'languages', 'marc_languages')
+         transform(strip: true, downcase: true), translation_map('not_found', 'languages', 'marc_languages')
```

Comment:
same here.

Reply:
that doesn't seem to work. Its not downcasing the values as far as I can tell. Any idea what's going wrong?

Reply:
Ah! We need this merged first: https://github.com/sul-dlss/traject_plus/pull/29 @justinlittman can you take a look?

Reply:
Done.

Reply:
Okay, next step would be to cut a 1.1.0 release of traject_plus and then update dlme-transform.

Reply:
@jacobthill those steps have been taken.  Can you pull the latest dlme-transform code, rebuild the container and see if this would work now?

Reply:
Yep, I’m on site this week so I’ll do it when I get to campus